src/pass.rs

```diff
@@ -43,6 +43,8 @@ pub type PasswordStoreType = Arc<Mutex<PasswordStore>>;
 
 /// Represents a complete password store directory
 pub struct PasswordStore {
+    /// Name given to the store in a config file
+    name: String,
```

Comment:
Why does the PasswordStore need to know its name? Isn't it better if this is something that is part of the config?

Reply:
In order to render the list of stores that the user can choose between, we could use the path instead and not have a name at all, but since the current config data structure is two nested map's there kind of exists a name that's natural to expose to the user.

Reply:
I didn't mean we don't need a name for it, my point was that it doesn't seem it belongs in the PasswordStore struct, but in the config struct.

Reply:
If it's not in the PasswordStore struct we would need some kind of link between what row in the config struct corresponds to what object in the Vec of PasswordStore's. And also pass around both structures instead of just one to a hypothetical edit config function.